README.md

```diff
@@ -44,8 +44,11 @@ If there's an issue downloading the files, or you are offline, or you just want
 to use the cache anyway, you can pass the `--use-cache` flag.
 
 ## Additional tools
+<details><summary>Details</summary>
```

Comment:
## General change
I needed to update the `README.md` due to the inclusion now of `run.sh` (should have been there before, honestly). I took the time to hide in this collapsible 'details' box some utilities that we don't really use anymore.